src/modules/fancyzones/lib/fancyzones.rc

```diff
@@ -20,6 +20,7 @@ BEGIN
     IDS_SETTING_DESCRIPTION_USE_CURSORPOS_EDITOR_STARTUPSCREEN "Follow mouse cursor instead of focus when launching editor in a multi screen environment"
     IDS_SETTING_DESCRIPTION_APPLASTZONE_MOVEWINDOWS            "Move newly created windows to their last known zone"
     IDS_SETTING_DESCRIPTION_RESTORESIZE                        "Restore the original size of windows when unsnapping"
+    IDS_SETTING_DESCRIPTION_OPEN_WINDOW_ON_ACTIVE_MONITOR      "Open newly created windows on active monitor"
```

Comment:
`Move newly created windows to the current active monitor`

Reply:
Modified.